packages/elements/src/slider/elements/slider-marker.ts

```diff
@@ -0,0 +1,58 @@
+import { BasicElement, CSSResultGroup, TemplateResult, css, html } from '@refinitiv-ui/core';
+import { customElement } from '@refinitiv-ui/core/decorators/custom-element.js';
+import { property } from '@refinitiv-ui/core/decorators/property.js';
+
+import { VERSION } from '../../version.js';
+
+@customElement('ef-slider-marker')
+export class SliderMarker extends BasicElement {
+  /**
+   * Element version number
+   * @returns version number
+   */
+  static override get version(): string {
+    return VERSION;
+  }
+
+  static override get styles(): CSSResultGroup {
+    return css`
+      :host {
+        position: absolute;
+        z-index: 2;
+      }
+      [part='marker'] {
+        width: inherit;
+        height: inherit;
```

Comment:
I believe `100%` is our intention. Plus, `inherit` would not work as expected when parent width is not `100%`. https://stackoverflow.com/questions/9374509/width100-vs-widthinherit shows the issue clearly.
```suggestion
        width: 100%;
        height: 100%;
```